build.gradle

```diff
@@ -113,6 +113,10 @@ dependencies {
   // this is to force transitive dependencies to use 1.0.24 instead of
   // 1.0.23 as that is affected by CVE-2022-31684
   implementation "io.projectreactor.netty:reactor-netty-http:1.0.24"
+  runtimeOnly "org.postgresql:postgresql:42.5.0"
+  //  runtimeOnly "org.postgresql:r2dbc-postgresql"
```

Comment:
Does this comment still serve something? 🤔

Reply:
We should remove that, you're right